src/core/algorithms/md/hymd/lattice/md_lattice.cpp

```diff
@@ -752,7 +754,11 @@ void MdLattice::GetLevel(MdNode& cur_node, std::vector<MdVerificationMessenger>&
 auto MdLattice::GetLevel(std::size_t const level) -> std::vector<MdVerificationMessenger> {
     std::vector<MdVerificationMessenger> collected;
     MdLhs current_lhs(column_matches_size_);
-    GetLevel(md_root_, collected, current_lhs, 0, level);
+    if (no_levels_) {
```

Comment:
There is no need for this `bool`, [check if the function is empty](https://en.cppreference.com/w/cpp/utility/functional/function/operator_bool) instead